website/blog/2021-06-13-Hydra_1.1.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Hydra 1.1
+author: Omry Yadan
+author_title: Creator of Hydra
+author_url: https://github.com/omry
+author_image_url: https://graph.facebook.com/733244046/picture/?height=200&width=200
+tags: [Hydra, Release]
+image: https://hydra.cc/img/Hydra-Readme-logo2.svg
+---
+
+<p align="center"><img src="/img/Hydra-Readme-logo2.svg" alt="logo" width="70%" /></p>
+
+After many months and a lot of hard work by many people, Hydra 1.1 is finally out!  
+Hydra 1.1 comes with OmegaConf 2.1, which has its own share of awesome new features.
+
+<!--truncate-->
+
+This blog post highlights some of the most prominent features, check the release notes for a complete list of changes:
+- Hydra 1.1 [release notes](https://github.com/facebookresearch/hydra/releases/tag/v1.1.0)
+- OmegaConf 2.1 [release notes](https://github.com/omry/omegaconf/releases/tag/v2.1.0)
+
+### Major new features in Hydra 1.1
+- More powerful config composition
+  - Every config can now have a Defaults List
+  - Composition order of current config can be controlled via the `_self_` keyword in the Defaults List
+  - Support for composing multiple configs from the same config group
+  - Support for configuring the config search path from the primary config
+- Recursive instantiation
+- Experimental callbacks support
+- OmegaConf 2.1:
+  - Relative interpolations
+  - New OmegaConf interpolation grammar supporting nested interpolations and much more
+  - More powerful custom resolvers
+
+### More powerful config composition
+Config composition is the key area of improvement in Hydra 1.1.  
+The biggest change is support for a Defaults List in any config, and not just the primary config.
+
+This enables many new capabilities:
+- Any config can now "extend" other configs. This enables config files to be associated with a Structured Config schema and to extend other config files
+- A top level "experiment config" can now override the Defaults List as well as config values
+- Complex frameworks can now have their own Defaults List, reducing boilerplate
+
+Other related changes includes the ability to change the order a config is composed relative to config in its Defaults List by 
+adding `_self_` to the Defaults List and the ability to use multiple configs from the same config group.
+
+Learn more:
+- [The Defaults List](/docs/next/advanced/defaults_list)
+- [Extending configs](/docs/next/patterns/extending_configs)
+- [Structured Configs Schema](/docs/next/tutorials/structured_config/schema)
+- [Configuring Experiments](/docs/next/patterns/configuring_experiments)
+- [Select multiple configs from config group](/docs/next/patterns/select_multiple_configs_from_config_group)
```

Comment:
I tried adding an `.md` extension to these like in PR #1649, but yarn couldn't resolve the links.
I think it doesn't matter though because these links are absolute, so the relative link bug #1640 won't be a problem.

Reply:
Yeah, the blog is outside of the versioned tree so it behaves differently. I was not able to link to md files.